doctrine.html

```diff
@@ -149,7 +149,7 @@ <h2 id="no-one-paradigm">No one paradigm</h2>
 
       <p>Even the very foundation of the Active Record framework offends some purists. We’re mixing the logic needed for interfacing with the database directly with the business domain and logic. Such conflation of boundaries! Yes, because it proved to the practical way to skin a web-app cat that virtually always talks to a database of some sort to persist the state of the domain model.</p>
 
-      <p>To be so ideologically flexible is what enables Rails to tackle such a wide array of problems. Most individual paradigms do very well within a certain slice of the problem space, but becomes awkward or rigid when applied beyond its natural sphere of comfort. By applying many overlapping paradigms we cover the flanks and guard the rear. The final framework is far stronger and more capable than any individual paradigm would have allowed it to be.</p>
+      <p>To be so ideologically flexible is what enables Rails to tackle such a wide array of problems. Most individual paradigms do very well within a certain slice of the problem space, but become awkward or rigid when applied beyond its natural sphere of comfort. By applying many overlapping paradigms, we cover the flanks and guard the rear. The final framework is far stronger and more capable than any individual paradigm would have allowed it to be.</p>
```

Comment:
This changes the meaning, doesn't it. Originally the subject is "the thing", "the thing, the experience, becomes ackward", while this change makes the paradigms the subject. Let me /cc @dhh, this is a very personal document and the original intent has to be preserved.